website/Makefile

```diff
@@ -18,7 +18,8 @@ deps:
 		if [ "$$file" = "/_index.md" -o "$$base_name" = "README" ]; then \
 			continue; \
 		fi; \
-		text_to_prepend="---\ntitle: \"$$base_name\"\n---\n"; \
+		title_case=$$(echo "$$base_name" | sed -e 's/-/ /g' -e 's/\b\(.\)/\u\1/g'); \
```

Comment:
<img width="333" alt="Screenshot 2024-06-20 at 12 55 43 PM" src="https://github.com/cncf/tag-security/assets/21176439/0e174974-39c1-489d-b64d-30ab6f661867">

Try this to cover edge cases:

```suggestion
		title_case=$$(echo "$$base_name" | sed -e 's/-/ /g' | awk '{for(i=1;i<=NF;i++) if($$i !~ /^[A-Z]/) $$i=toupper(substr($$i,1,1)) substr($$i,2)}1'); \
```